content/en/docs/concepts/configuration/configmap.md

```diff
@@ -221,6 +221,60 @@ in a Pod:
    in the specified environment variables.
 
 This is an example of defining a ConfigMap as a pod environment variable:
+
+The following ConfigMap (myconfigmap.yaml) stores two properties: username and access_level:
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: myconfigmap
+data:
+  username: "k8s-admin"
+  access_level: "1"
+```
+The following command will create the ConfigMap object:
+```shell
+$ kubectl apply -f myconfigmap.yaml
+configmap/myconfigmap created
+```
```

Comment:
We [don't include the command prompt](https://kubernetes.io/docs/contribute/style/style-guide/#don-t-include-the-command-prompt)